### ISSUE TITLE
Contributor License Agreement link in Readme is broken

### ISSUE THREAD

**Issue by Zocker1999NET** [schwabe/ics-openvpn#1750]
https://github.com/schwabe/ics-openvpn/blob/ad7dfd597299f6cdf339ca924621733d3edecb0d/README.md?plain=1#L62

The link in that line on the current master seems broken. This link now refers to a review of a TV series, written in French.

I saw that this was already mentioned, see https://github.com/schwabe/ics-openvpn/issues/1698#issuecomment-2025049475, but it seemed to me that this was ignored by accident. I saw no other issue still open mentioning that.

I have not found a working link or reference to the CLA in doc/README.txt nor in ./CONTRIBUTING.

**Comment by Benjamin-Loison** [comment]
As far as I understand quickly reading [DamageLabs/clahub/issues/174#issuecomment-624285851](https://github.com/DamageLabs/clahub/issues/174#issuecomment-624285851) and [DamageLabs/clahub/issues/111](https://github.com/DamageLabs/clahub/issues/111). It seems that because of a lack of maintenance clahub.com owner lost this domain name (and its data?).

https://cla-hub.io seems to be the new domain, according to: [DamageLabs/clahub/commit/137c7c8af145b4d9eeda6b64de4f49650f40192c](https://github.com/DamageLabs/clahub/commit/137c7c8af145b4d9eeda6b64de4f49650f40192c)

**Comment by schwabe** [comment]
Yeah. I switched in the meantime to an alternative CLA generator and put the required files directly into this repository so that this does not happen again. But thanks for pointing out a good alternative.

**Comment by mehdi2007mp** [comment]
Hello, how are you? Yes, I received it. I checked it a lot, but I still
couldn't fix it.

در تاریخ سه‌شنبه ۱۹ مه ۲۰۲۶،‏ ۲۲:۳۵ Benjamin Loison <
***@***.***> نوشت:
